src/data/text/move_names.h

```diff
@@ -354,5 +354,6 @@ const u8 gMoveNames[MOVES_COUNT][MOVE_NAME_LENGTH + 1] = {
     [MOVE_WATER_PULSE]   = _("WATER PULSE"),
     [MOVE_DOOM_DESIRE]   = _("DOOM DESIRE"),
     [MOVE_PSYCHO_BOOST]  = _("PSYCHO BOOST"),
-    [MOVE_HEART_SWAP]    = _("HEART SWAP")
+    [MOVE_HEART_SWAP]    = _("HEART SWAP"),
+    [MOVE_SUBSTITUTE_TEACHER]    = _("SUBSTITUTE")
```

Comment:
Is Substitute Teacher intended to just be named Substitute?

Reply:
Yes. Just Celia things :D